pkgs/servers/http/jetty/default.nix

```diff
@@ -2,11 +2,10 @@
 
 stdenv.mkDerivation rec {
   pname = "jetty";
-  version = "9.4.35.v20201120";
+  version = "9.4.36.v20210114";
   src = fetchurl {
     url = "https://repo1.maven.org/maven2/org/eclipse/jetty/jetty-distribution/${version}/jetty-distribution-${version}.tar.gz";
-    name = "jetty-distribution-${version}.tar.gz";
-    sha256 = "1cpdrqz6wi7fd228lh4ijs82jq51qr5aym6vff464w3y2sd0jbav";
+    sha256 = "1bsqxzjcdgdg2qdgc64pvrimkn9j2di2s3prlgdpbwi566744q54";
   };
 
   phases = [ "unpackPhase" "installPhase" ];
```

Comment:
phases should not be overwritten because it breaks many hooks. Phases which should not run should be disabled.

Reply:
Do I set `buildPhase = null`?
There's nothing about it in https://nixos.org/manual/nixpkgs/stable/#ssec-controlling-phases.

Reply:
``doBuild = false;``

Reply:
done